src/manifest-chrome.json

```diff
@@ -1,19 +1,26 @@
 {
-	"manifest_version": 3,
-	"name": "Datahub Link generator",
-	"version": "3.0",
-	"description": "A browser extension to generate Datahub links directly from GitHub",
-	"permissions": [
-		"tabs"
-	],
-	"background": {
-		"service_worker": "background.js"
-	},
-	"action": {
-		"default_icon": {
-			"32": "icon_32.png"
-		},
-		"default_title": "Generate Datahub link",
-		"default_popup": "popup.html"
-	}
+    "manifest_version": 3,
+    "name": "DataHub Link Generator",
+    "version": "1.7.1",
+    "description": "Generate shareable Datahub links directly from GitHub",
+    "icons": {
+        "32": "icon_32.png"
```

Comment:
The icon is mentioned here, but this commit deletes the icon files.

Reply:
Added the icons back!